### ISSUE TITLE
Check early for output directory

### ISSUE THREAD

**Issue by m-pauper** [vdemichev/DiaNN#720]
I think it would be very useful that the program checks whether the output locations exist and are writable at the very beginning. As it is, after a 6-days-long processing, it just did not write the output because the output directory had a typo.

Thanks!

**Comment by vdemichev** [comment]
Agree. Can also reanalyse with 'Use existing .quant files' (will be instantaneous) and specifying the new output location.
